sdf/quickstart.mdx

```diff
@@ -161,4 +160,7 @@ fluvio topic delete words
 [Composable Dataflows]: compositions/overview.mdx
 [dataflow.yaml]: concepts/dataflow-yaml.mdx
 [github]: https://github.com/infinyon/stateful-dataflows-examples/
-[Example Workflows in github]: https://github.com/infinyon/stateful-dataflows-examples
\ No newline at end of file
+[Example Workflows in github]: https://github.com/infinyon/stateful-dataflows-examples
+[Fluvio Cluster]: /docs/fluvio/quickstart#start-a-cluster
+[Dataflow File]: /sdf/concepts/dataflow-yaml
```

Comment:
this is duplicate of link above

Reply:
![Screenshot 2024-08-22 at 11 57 20 AM](https://github.com/user-attachments/assets/1afe6110-9689-49fd-9666-391011177fcb)

Mmm seems like we need to change the aliases to match both instead